app/helpers/solidus_viabill/api/checkout_helper.rb

```diff
@@ -6,7 +6,7 @@ module CheckoutHelper
       VIABILL_PROTOCOL = '3.1'
       VIABILL_STATUS = %w[CANCELLED APPROVED REJECTED].freeze
       def build_checkout_request_body(order, payment_method_id, frontend)
-        gateway = SolidusViabill::Gateway.new
+        gateway = SolidusViabill::Gateway.new({})
```

Comment:
Maybe we should create the gateway here with the correct arguments so that we can use the `api_key` and the `secret_key` from the gateway instead of these lines
```rb
apikey: payment_method.preferences[:viabill_api_key],
```
```rb
payment_method.preferences[:viabill_secret_key],
```